### ISSUE TITLE
Fixed fetal head flexion

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11567]
**Preferred term label:**
Fixed fetal head flexion

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID 27987045

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:253290

**Your nano-attribution (ORCID)**
0000-0001-9947-035X

**Comment by pnrobinson** [comment]
@poratshay 
Adding new term: 
- ID: HP:6001473
- label: Fixed fetal head flexion
- parents: [Fetal head anomaly (HP:0034245)](https://hpo.jax.org/app/browse/term/HP:0034245)
- pmid: PMID:27987045
- def: An unchanging bending forward of the fetal head in the uterus as visualized by prenatal sonography. This sign can be defined as a lack of change in this position during the 30 min of observation	. Additionally, identical head flexion is observable several times at intervals.
- comment: This feature has been observed in fetuses with Multiple pterygium syndrome, lethal type.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11567
- synonyms:
